### PR TITLE
Improve helm/kustomize separation

### PR DESCRIPTION
**Pull request by devcatalin**
This PR...

## Changes

- improved Icon atom component
- added Helm & Kustomize icons
- moved Helm Charts and Kustomizations sections to their own left menu pane
- rewritten the logic to scroll items into view, it works better and now it shouldn't scroll if there are already items highlighted in the view
- refactored left menu button and icon components
- added customizable text for sections before initialization

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/navsections/HelmChartSectionBlueprint/HelmChartSectionBlueprint.ts [hunk 1]
```diff
@@ -53,12 +61,17 @@ const HelmChartSectionBlueprint: SectionBlueprint<HelmValuesFile, HelmChartScope
...
       return rawItems.length > 0;
     },
+    isEmpty: (scope, rawItems) => {
+      return scope.isFolderOpen && rawItems.length === 0;
+    },
+    shouldBeVisibleBeforeInitialized: true,
+  },
+  customization: {
+    emptyDisplay: {component: HelmChartSectionEmptyDisplay},
+    beforeInitializationText: 'Get started by browsing a folder in the File Explorer.',
```
Comment (olensmar) [inline src/navsections/HelmChartSectionBlueprint/HelmChartSectionBlueprint.ts:74]: this needs to show something else if the current folder has no Helm charts?

File: src/navsections/KustomizationSectionBlueprint/KustomizationSectionBlueprint.ts [hunk 2]
```diff
@@ -51,29 +58,28 @@ const KustomizationSectionBlueprint: SectionBlueprint<K8sResource, Kustomization
...
+    isEmpty: (scope, rawItems) => {
+      return scope.isFolderOpen && rawItems.length === 0;
+    },
+    shouldBeVisibleBeforeInitialized: true,
+  },
+  customization: {
+    emptyDisplay: {
+      component: KustomizationSectionEmptyDisplay,
+    },
+    beforeInitializationText: 'Get started by browsing a folder in the File Explorer.',
```
Comment (olensmar) [inline src/navsections/KustomizationSectionBlueprint/KustomizationSectionBlueprint.ts:73]: this needs to show something else if the current project has no Kustomizations?

Comment (devcatalin) [reply src/navsections/KustomizationSectionBlueprint/KustomizationSectionBlueprint.ts:73]: Yes, it does, the text for when there are no kustomizations in the folder is in the KustomizationSectionEmptyDisplay component

In a future PR when I start adding tests, I will refactor the customization interface to either allow a string or a component on some of it's options. For now the "emptyDisplay" needs a component

**Review by olensmar — APPROVED** [review]

**Comment by olensmar** [comment]
Looks great!

**Pull request merged**